src/styled/layouts/basic-layout.ts

```diff
@@ -14,7 +14,7 @@ export const Root = styled(Box, {
   display: "grid",
   gridTemplateRows: "auto auto auto 1fr",
   backgroundColor: theme.palette.background.paper,
-  backgroundImage: `url(${theme.backgroundImage.backgroundImagePath})`,
+  backgroundImage: `url(${process.env.REACT_APP_THEME_BACKGROUNDIMAGE_PATH})`,
```

Comment:
Move all the envs to config.ts and make functions to use them. This applies to whole pr, replace all "process.env..." to be a getter method.